core/src/main/java/org/bitcoinj/core/PeerAddress.java

```diff
@@ -351,10 +353,16 @@ public BigInteger getServices() {
         return services;
     }
 
-    public long getTime() {
+    public Optional<Instant> getTimeInstant() {
```

Comment:
This should have JavaDoc explaining under what circumstances (old peer) the `Instant` is not available.

Reply:
You're right. I've just added that.